source/3rd-party/clipp/include/clipp.hpp

```diff
@@ -0,0 +1,6408 @@
+/*****************************************************************************
+ *  ___  _    _   ___ ___
+ * |  _|| |  | | | _ \ _ \   CLIPP - command line interfaces for modern C++
+ * | |_ | |_ | | |  _/  _/   version 1.1.0
```

Comment:
Latest version is 1.2.3: https://github.com/muellan/clipp/blob/master/include/clipp.h

Reply:
done